colorcet/plotting.py

```diff
@@ -19,6 +19,8 @@ def swatch(name, cmap=None, bounds=None, array=array, **kwargs):
     if bounds is None:
         bounds = (0, 0, 256, 1)
 
+    cmap = list(cmap) if cmap is not None else None
+
```

Comment:
This line may need to move under "if 'bokeh' in backends and not overwrite the main cmap argument, because this code is meant to work with both Bokeh and Matplotlib. In Matplotlib a cmap is not list-like; it's usually an instantiated class.  So this change might have broken `swatch` for Matplotlib colormaps...

Reply:
Moving it in there won't be sufficient, it should probably just explicitly check for tuple types and then cast it.

Reply:
I agree, that would be the fully safe thing to do.